app/src/javascripts/devextreme.js

```diff
@@ -9,6 +9,4 @@ window.Globalize = Globalize;
 import '../../assets/javascripts/data_table';
 import '../../assets/javascripts/data_table_templates';
 import '../../assets/javascripts/master_detail';
-// When we have fixed the issue with "Fixed columns break sorting", then we can reference devextreme npm packages
-// import 'devextreme/dist/js/dx.all.debug';
-import '../../assets/javascripts/dx.webappjs';
```

Comment:
dont we need to include this anymore?

Reply:
Nope. We only had it there because we had a "fix" in there to sort out the breaking sorting. The reason the sorting was breaking was because of a jQuery Extension we wrote that was not returning the value when it should have. I've fixed that now. So we can now reference the npm package instead.